packages/transformers/webext/src/schema.js

```diff
@@ -0,0 +1,402 @@
+// @flow strict-local
+import type {SchemaEntity} from '@parcel/utils';
+import {extname} from 'path';
+
+// Non-comprehensive, but should cover the vast majority of users
+// SVGs not supported in WebKit/Blink
+const IMG_EXT = new Set(['jpg', 'jpeg', 'png', 'svg', 'webp', 'gif']);
+
+const validateVersion = (ver: string): ?string => {
+  const parts = ver.split('.', 5);
+  if (parts.length == 5) return 'Extension versions to have at most three dots';
+  // greater than to allow for NaN to fail
+  if (parts.every(part => +part[0] > 0 && +part < 65536)) return;
+  return 'Extension versions must be dot-separated integers between 0 and 65535';
+};
+
+const validateImage = (name: string): ?string => {
+  const ext = extname(name).slice(1);
+  if (!IMG_EXT.has(ext))
+    return `Extensions do not support ${ext} images and icons`;
+};
+
+const icons: SchemaEntity = {
+  type: 'object',
+  properties: {},
+  additionalProperties: {
+    type: 'string',
+    __validate: validateImage,
```

Comment:
I don't think this should be that strict. Maybe you could even have a pipeline that converts between image formats. In that case, these "invalid" formats should be allowed here.

Reply:
I hadn't considered other transformers. Will fix.